### ISSUE TITLE
Implement Reflection

### ISSUE THREAD

**Issue by stephen-dahl** [malijs/mali#50]
Hi, good work with this so far!

Mali should support reflection once it is implemented in the core lib https://github.com/grpc/grpc/issues/2568

Or sooner if you dont need them :)

**Comment by bojand** [comment]
Yes this would be useful, and will implement once it's in the core grpc library.

**Comment by kashifmin** [comment]
Has this been implemented yet ? @bojand

**Comment by bojand** [comment]
Hello, thank you for inquiring about this issue. This is not implemented yet as the core grpc Node.js module does not support reflection yet.

**Comment by lalomartins** [comment]
Does core actually need to support it first? Python and C++ have reflection as separate packages, and AIUI the js structures created either by dynamic or static should already have all the metadata (descriptors) needed by reflection. The ticket in the core lib mentions it can be done as a separate library.

Bonus: that means Mali can probably also have it as an optional add-on `mali-reflection`, so you only get it if you specifically want it.

**Comment by ansarizafar** [comment]
Any there any new update on this issue?

**Comment by ansarizafar** [comment]
It seems that Reflection can be implemented without support in core grpc library   https://github.com/grpc/grpc-node/issues/79#issuecomment-491699115
